packages/Php/tests/Rector/FuncCall/ParseStrWithResultArgumentRector/Fixture/already_set.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Php\Tests\Rector\FuncCall\ParseStrWithResultArgumentRector\Fixture;
+
+class AlreadySet
+{
+    public function run()
+    {
+        $query = 'bla';
+        parse_str($query, $data);
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Php\Tests\Rector\FuncCall\ParseStrWithResultArgumentRector\Fixture;
+
+class AlreadySet
+{
+    public function run()
+    {
+        $query = 'bla';
+        parse_str($query, $data);
+    }
+}
+
+?>
```

Comment:
If the code should have no change, you can keep only the first part (above `----`):

```diff
<?php

namespace Rector\Php\Tests\Rector\FuncCall\ParseStrWithResultArgumentRector\Fixture;

class AlreadySet
{
    public function run()
    {
        $query = 'bla';
        parse_str($query, $data);
    }
}
?>
------
-<?php
-
-namespace Rector\Php\Tests\Rector\FuncCall\ParseStrWithResultArgumentRector\Fixture;
-
-class AlreadySet
-{
-    public function run()
-    {
-        $query = 'bla';
-        parse_str($query, $data);
-    }
-}
-?>
```

Reply:
Also, the standard here is  to prefix these test cases with `skip_`.
That way we can group them visually by *fix* and *skip* groups

Here `skip_already_set.php.inc`.